### PR TITLE
Format the code and remove warnings

### PR DESCRIPTION
**Pull request by jonnxu**
## What is the purpose of the change

Format the code and remove warnings

## Brief changelog

MQClientErrorContainer.cpp
MQClientErrorContainer.h
.........

## Verifying this change

success

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when a cross-module dependency exists.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: include/CErrorMessage.h [hunk 1]
```diff
@@ -24,10 +23,10 @@
...
-	ROCKETMQCLIENT_API const char *GetLatestErrorMessage();  // Return the last error message
+
+ROCKETMQCLIENT_API const char* GetLatestErrorMessage();  // Return the last error message
 
 #ifdef __cplusplus
 };
 #endif
-#endif //__C_CLIENT_ERROR_H__
\ No newline at end of file
+#endif  //__C_CLIENT_ERROR_H__
```
Comment (jovany-wang) [inline include/CErrorMessage.h]: Append a new blank line at the end of this file.

File: include/CErrorMessage.h [hunk 2]
```diff
@@ -24,10 +23,10 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-	 
-	ROCKETMQCLIENT_API const char *GetLatestErrorMessage();  // Return the last error message
+
+ROCKETMQCLIENT_API const char* GetLatestErrorMessage();  // Return the last error message
```
Comment (jovany-wang) [inline include/CErrorMessage.h]: Why change this line? does complier complain about something with this line?

File: src/common/MQClientErrorContainer.h [hunk 3]
```diff
@@ -1,43 +1,41 @@
...
-		static MQClientErrorContainer * instance();
+class MQClientErrorContainer {
+ public:
+  static MQClientErrorContainer* instance();
 
-		void setErr(std::string str);
+  void setErr(std::string str);
 
-		std::string getErr();
+  std::string getErr();
```
Comment (jovany-wang) [inline src/common/MQClientErrorContainer.h]: `  std::string getErr() const;
`

File: src/transport/TcpRemotingClient.cpp [hunk 4]
```diff
@@ -373,7 +373,7 @@ std::shared_ptr<TcpTransport> TcpRemotingClient::CreateNameServerTransport(bool
       m_namesrvAddrChoosed.clear();
   }
 
-  for (int i = 0; i < m_namesrvAddrList.size(); i++) {
+  for (unsigned i = 0; i < m_namesrvAddrList.size(); i++) {
```
Comment (ifplusor) [inline src/transport/TcpRemotingClient.cpp]: size_t is better

**Comment by jonnxu** [comment]
see #172 
https://github.com/apache/rocketmq-client-cpp/pull/172

**Comment by ShannonDing** [comment]
it seems duplicated with #172 , could you please check and close one of them.

**Pull request closed without merging**